test/framework/src/main/java/org/elasticsearch/test/VersionUtils.java

```diff
@@ -159,6 +156,19 @@ public static Version getPreviousVersion(Version version) {
         throw new IllegalArgumentException("couldn't find any released versions before [" + version + "]");
     }
 
+    /**
+     * Get the released version after {@code version}.
+     */
+    public static Version getNextVersion(Version version) {
+        for (int i = 0; i < ALL_VERSIONS.size(); i++) {
```

Comment:
This should loop over `RELEASED_VERSIONS`.

Reply:
Why is that? I wanted it to test 6.3 and 6.4 (which is the only version which is min compatible with master). I will fix the java docs though - copy paste left over.

Reply:
compatibleFutureVersion works for me. changed.